devito/finite_difference.py

```diff
@@ -137,18 +137,23 @@ def first_derivative(*args, **kwargs):
     dim = kwargs.get('dim', x)
     diff = kwargs.get('diff', h)
     order = kwargs.get('order', 1)
-    side = kwargs.get('side', 1)
+    side = kwargs.get('side', 0)
     deriv = 0
     # Stencil positions for non-symmetric cross-derivatives with symmetric averaging
     if side == 1:
         ind = [(dim + i * diff) for i in range(-int(order / 2) + 1 - (order % 2),
                                                int((order + 1) / 2) + 2 - (order % 2))]
-    else:
+    elif side == -1:
         ind = [(dim - i * diff) for i in range(-int(order / 2) + 1 - (order % 2),
                                                int((order + 1) / 2) + 2 - (order % 2))]
+    else:
```

Comment:
is this the case ``side==0`` ?
if that's the case, can you change the value of side to 1 here (again: better off if no magic numbers are used), rather than below?